manuscript/manuscript.md

```diff
@@ -70,14 +76,38 @@ Since analytical and data processing procedures may improve or otherwise change
 * CSV for tabular data
 * HDF5 for (??)
 * ?? for images
 * (help me out here folks, it's been a long week)
 and examples of closed formats include XLSX, DICOM, (again need more examples). At a minimum, data being stored for archival purposes should be stored in open formats, even if day-to-day processing uses closed formats.
 
-# Rule 5: Rule {-}
+# Rule 5: Rule {Data should be uniquely identifiable}
+
+To aid reproducibility, the data used in a scientific publication should be uniquely identifiable.
+Ideally, datasets should have a unique identifier such as a Document Object Identifier (DOI), Archival Resource Key (ARK), or a Persistant URL (PURL).
+An increasing number of online services, such as [Figshare](http://figshare.com/) are able to provide these.
+
+Datasets may evolve over time and in order to distinguish between different versions of the same data *Semantic versioning* should be used.
+An example of this can be seen in the CellPack datasets [@johnson2014cellpack].
+
+A semantic version number takes the form: `Major.Minor.Patch`, e.g. `0.2.7`.
+The **major version** numbers should incremented (or *bumped*) when a dataset scheme has been updated, or some other change is made that is not compatible with previous versions of the data with the same major version number.
+This might mean that an experiment using version `1.0.0` of the dataset could not be run on version `2.0.0` without making some changes to the data analysis.
+The **minor version** should be bumped when a change has been made which is compatible with older versions of the data with the same Major version.
+This means that any analysis that can be performed on version `1.0.0` of the data should be repeatable with version `1.1.0` of the data.
+The **patch version** number should be bumped when typos or bugs have been fixed.
+For example version `1.0.1` of a dataset may fix a typo in version `1.0.0`.
+
+A common, but less semantically dense, alternative to semantic versioning is the use of date stamps as file or directory names to describe when the data was produced.
+This is problematic, as formats for date stamps vary regionally, in ways that can render the date stamp ambiguous.
+For example, the date `01-02-2015` can mean 02 Jan 2015 (US format) or 01 Feb 2015 (UK format), without metadata to say how the date stamp is formatted the reader cannot tell which date this is.
+
+To avoid regional ambiguities, it is wise to use the ISO 8601 standard, which mandates the date format `YYYY-MM-DD` (i.e. from largest time unit to smallest).
```

Comment:
Given the scope (simple) I think this line (107) is sufficient to cover the detail in lines 104, 105 

Also, perhaps present date first, then mention semantic versioning as an alternative.